identity-service/src/routes/authentication.js

```diff
@@ -30,7 +31,19 @@ module.exports = function (app) {
 
     if (body && body.iv && body.cipherText && body.lookupKey) {
       try {
-        await models.Authentication.create({ iv: body.iv, cipherText: body.cipherText, lookupKey: body.lookupKey })
+        await sequelize.transaction(function (t) {
```

Comment:
you can assign this to a variable and use async/await like https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/routes/nodeSync.js#L30

Reply:
I'll change it! Just saw there's the extra `transaction.commit()` I missed on first glance - that makes it much better! was wondering when the transaction would finish otherwise.